pkg/syncer/storagepool/intended_state.go

```diff
@@ -366,15 +369,15 @@ func (c *SpController) updateIntendedState(ctx context.Context, dsMoid string, d
 func (c *SpController) updateVsanSnaIntendedState(ctx context.Context, vsanState *intendedState,
 	validStoragePoolNames map[string]bool, scWatchCntlr *StorageClassWatch) error {
 	log := logger.GetLogger(ctx)
-	vsanHostCapacities, err := c.getVsanHostCapacities(ctx)
-	if err != nil {
-		log.Errorf("Error encountered fetching vSAN SNA Host capacities. Err: %v", err)
-		return err
-	}
 	vsanNodes := make(map[string]bool)
 	for _, vsanNode := range vsanState.nodes {
 		vsanNodes[vsanNode] = true
 	}
+	vsanHostCapacities, hostCapacityErr := c.getVsanHostCapacities(ctx)
+	if hostCapacityErr != nil {
+		log.Errorf("Error encountered fetching vSAN SNA Host capacities. Err: %v", hostCapacityErr)
+	}
+
```

Comment:
Is there any significance is moving this code down?

Reply:
This was done to improve the readability of code slightly. To ease reader in figuring out what were the steps performed before returning hostCapacityErr, if any.